.github/workflows/qa-checks.yml

```diff
@@ -0,0 +1,30 @@
+name: OSP Web QA Checks
+
+on: [push, pull_request]
+
+jobs:
+  run-linters:
+    name: Linters and Formatters
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      # ESLint and Prettier must be in `package.json`
+      - name: Install Node.js dependencies
+        run: npm install
+
+      - name: Linters and Formatters Check
+        uses: wearerequired/lint-action@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          # auto_fix: true
```

Comment:
if this is not required you can remove this line. @jalajcodes

Reply:
Done!